### ISSUE TITLE
Hi, I have a question about this code?

### ISSUE THREAD

**Issue by BCWang93** [L1aoXingyu/fcn.pytorch#4]
When I run this code FCN32s to training voc2012 dataset,I get some error like this:
the origin image size is like this:
torch.Size([1, 3, 333, 500])#image
torch.Size([1, 333, 500])#label
but the out size of the net is this:
torch.Size([1, 21, 141, 301])#output of net
So,I think this have some problem,the size of (h,w) is not same as the origin!Can you help me to solve this?Thanks!
